packages/zipkin/index.d.ts

```diff
@@ -306,7 +306,13 @@ declare namespace zipkin {
 
   namespace Instrumentation {
     class HttpServer {
-      constructor(args: { tracer: Tracer, port: number });
+      constructor(args: {
+        tracer: Tracer,
+        port: number,
+        serviceName?: string,
+        host?: string,
+        serverTags?: {[key: string]: string}
```

Comment:
do we want to qualify only server or just default? ex what about message consumer?

Reply:
I would keep it as serverTags because these are only tags for server.

Reply:
my intention was to hint you towards Tracer.defaultTags vs sporadically adding to every type of instrumentation.

The rationale is that this is default, there's no input parameter.. the more apis we add, the more we have to carry forward. I don't believe that a parameterless setting like this will serve us better than Tracer.defaultTags.

Reply:
so literally I was thinking you pass the tracer, you copy or take a reference to its default tags. then you get a single place to add and manage tags from. if there was a parameter type (ex request object etc), this wouldn't be sensible, but considering there is no scope here, and no example of tag that needs to know if it is a server or not, the simplest solution seems to be Tracer.defaultTags imho